lib/protobuf/varint_pure.rb

```diff
@@ -1,12 +1,12 @@
 module Protobuf
   module VarintPure
     def decode(stream)
-      value = index = 0
+      value = shift = 0
       begin
         byte = stream.readbyte
-        value |= (byte & 0x7f) << (7 * index)
```

Comment:
Bikeshedding a little, but honestly I prefer the hex. Hex makes it easier to picture the individual bits in my head.